src/main/java/org/truffleruby/language/arguments/ProfileArgumentNode.java

```diff
@@ -73,4 +75,9 @@ protected static boolean exactCompare(double a, double b) {
         return Double.doubleToRawLongBits(a) == Double.doubleToRawLongBits(b);
     }
 
+    @Override
+    public String toString() {
+        return getChild().toString();
+    }
```

Comment:
I'm not sure about these new `toString` methods - some of them seem bad - like this one, and they litter up the code base with what is essentially support for printf debugging.

Reply:
It's actually useful for me when using a Java debugger so I don't need to expand 3 times to see which index is read and which variable written.
What's the problem with this one? Do we want to show the profiling? I think it's just a transparent node just passing the value in the AST.

Reply:
I think it's going to be confusing to look at a `ProfileArgumentNode` and have it show up as something else entirely with no way to know what's going on.

Reply:
Would you be fine with something like `"Profiled(" + getChild() + ")"` ?

Reply:
I think that will save pain in the future, yes. Could shorten it to `P()` if you wanted.

Reply:
Fixed.